.env

```diff
@@ -17,7 +17,7 @@ SOLR_VERSION=v1.0.0
 IIIF_IMAGE_VERSION=v1.0.0
 POSTGRES_VERSION=v1.0.0
 BLACKLIGHT_VERSION=v1.6.0
-MANAGEMENT_VERSION=v1.7.2
+MANAGEMENT_VERSION=v1.8.0
```

Comment:
is it generally the practice to bump the version while doing code changes or do we release separately? curious what the pattern has been here

Reply:
I think the pattern has not been consistent, but I think doing it when we do the release would be better. Maybe we could add it to the readme on deployment in a separate PR, I can take it out of this PR.